hikari/api/rest.py

```diff
@@ -121,9 +121,15 @@ async def fetch_channel(
             If you are missing the `READ_MESSAGES` permission in the channel.
         hikari.errors.NotFoundError
             If the channel is not found.
+        hikari.errors.RateLimitTooLongError
+            Raised in the event that a rate limit occurs that is
+            longer than `max_rate_limit` when making a request.
+        hikari.errors.RateLimitTooLongError
+            Raised in the event that a rate limit occurs that is
+            longer than `max_rate_limit` when making a request.
         hikari.errors.RateLimitedError
             Usually, Hikari will handle and retry on hitting
-            rate-limits automatically. This includes bucket-specific
+            rate-limits automatically. This includes most bucket-specific
```

Comment:
Shouldnt this change also be made on all the other endpoints documentation for `hikari.errors.RateLimitedError`?